### ISSUE TITLE
Danger alert:Could not find the resource that you are looking for

### ISSUE THREAD

**Issue by jwklijnsma** [keycloak/keycloak#34585]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

![Screenshot 2024-11-02 at 09 27 12](https://github.com/user-attachments/assets/36f79fe0-4eb3-47eb-8a41-af96e932f2c6)
When refresh page /auth/admin/master/console/#/test/users/ multie time you get the error Danger alert:Could not find the resource that you are looking for but.

no logging in the database are keycloak are nginx-ingress why is page is comening 

### Version

kc26

### Regression

- [X] The issue is a regression

### Expected behavior

When refresh page /auth/admin/master/console/#/test/users/ multie time you get the error Danger alert:Could not find the resource that you are looking for but 

### Actual behavior

just refresh page  /auth/admin/master/console/#/test/users/ multie time you get the error Danger alert:Could not find the resource that you are looking for but can not find any issue with proxy

### How to Reproduce?

deploy keycloak kc22 are higher with postgress database cnpg

**Comment by jwklijnsma** [comment]
so this request is not good.

Request URL:

https:///auth/realms/master/protocol/openid-connect/login-status-iframe.html/init?client_id=security-admin-console&origin=https%3A%2F%2F

Request Method:

GET

Status Code:

204 No Content

Remote Address:


Referrer Policy:

no-referrer

referrer-policy:

no-referrer

strict-transport-security:

max-age=31536000; includeSubDomains

x-content-type-options:

nosniff

x-xss-protection:

1; mode=block

:authority:


:method:

GET

:path:

/auth/realms/master/protocol/openid-connect/login-status-iframe.html/init?client_id=security-admin-console&origin=https%3A%2F%2F

:scheme:

https

accept:

*/*

accept-encoding:

gzip, deflate, br, zstd

accept-language:

en-US,en;q=0.9,nl;q=0.8,fil;q=0.7,fy;q=0.6,de;q=0.5,it;q=0.4

cookie:

dnt:

1

priority:

u=1, i

sec-ch-ua:

"Chromium";v="130", "Google Chrome";v="130", "Not?A_Brand";v="99"

sec-ch-ua-mobile:

?0

sec-ch-ua-platform:

"macOS"

sec-fetch-dest:

empty

sec-fetch-mode:

cors

sec-fetch-site:

same-origin

user-agent:

Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36

**Comment by ssilvert** [comment]
@jwklijnsma I can't reproduce this.  

You are saying that you just refresh the user list multiple times and it eventually fails?

How many users do you have?  Are you running the latest version of Keycloak 26?

**Comment by jwklijnsma** [comment]
one time 2 second order time 10 second just one user will get more input i can reproduce on 4 testing systemen.

**Comment by ssilvert** [comment]
> one time 2 second order time 10 second just one user will get more input i can reproduce on 4 testing systemen.

I don't understand what you just wrote.  Can you answer my questions?

**Comment by jwklijnsma** [comment]
only 1 user in keycloak now yes i am running keycloak 26.

i have idee that is the issue 

            - name: KC_HOSTNAME_URL
              value: sso-***
            - name: KC_HOSTNAME_ADMIN_URL
              value: admin.***
            - name: KC_HOSTNAME
              value: https://***/auth
            - name: KC_HOSTNAME_ADMIN
              value: https://***/auth

order people say you need to change  set responseMode to query instead of fragment(default value)

**Comment by ssilvert** [comment]
> only 1 user in keycloak now yes i am running keycloak 26.
> 

Is it Keycloak 26.0.5?

**Comment by jwklijnsma** [comment]
Yes that is like that but i have nginx-ingress in frond of it and admin url is change.
Can it maybe issue with header is broken

**Comment by shawkins** [comment]
```
       - name: KC_HOSTNAME_URL
          value: sso-***
        - name: KC_HOSTNAME_ADMIN_URL
          value: admin.***
        - name: KC_HOSTNAME
          value: https://***/auth
        - name: KC_HOSTNAME_ADMIN
          value: https://***/auth
```

On KC26+ only KC_HOSTNAME and KC_HOSTNAME_ADMIN are used. What have you set proxy-headers to? And/or can you just provide the show-config output?

**Comment by jwklijnsma** [comment]
- name: KEYCLOAK_HTTP_RELATIVE_PATH
              value: /auth/
            - name: KC_SPI_ADMIN_REALM
              value: master
            - name: KC_LEGACY_OBSERVABILITY_INTERFACE
              value: 'true'
            - name: KC_HEALTH_ENABLED
              value: 'true'
            - name: KC_HOSTNAME_ADMIN
              value: https://******/auth/
            - name: KC_HOSTNAME
              value: https://*****/auth/
            - name: KC_FEATURES
              value: multi-site,persistent-user-sessions
            - name: KC_CACHE_REMOTE_HOST
              value: infinispan.****.svc
            - name: KC_CACHE_REMOTE_TLS_ENABLED
              value: 'false'
            - name: KC_REMOTE_STORE_PORT
              value: '11222'
            - name: KC_CACHE_REMOTE_USERNAME
              value: developer
            - name: KC_CACHE_REMOTE_PASSWORD
             
 KEYCLOAK_ADMIN: admin
  KEYCLOAK_CACHE_TYPE: local
  KEYCLOAK_ENABLE_HTTPS: 'true'
  KEYCLOAK_ENABLE_STATISTICS: 'true'
  KEYCLOAK_HTTPS_CERTIFICATE_FILE: /opt/keycloak/certs/tls.crt
  KEYCLOAK_HTTPS_CERTIFICATE_KEY_FILE: /opt/keycloak/certs/tls.key
  KEYCLOAK_HTTPS_PORT: '8443'
  KEYCLOAK_HTTPS_USE_PEM: 'true'
  KEYCLOAK_HTTP_PORT: '8080'
  KEYCLOAK_LOG_LEVEL: INFO
  KEYCLOAK_LOG_OUTPUT: default
  KEYCLOAK_PRODUCTION: 'true'
  KEYCLOAK_PROXY_HEADERS: ''

**Comment by shawkins** [comment]
What are the KEYCLOAK_ prefixed variables for? Those are not used by Keycloak.

I cannot reproduce this either in a typical setup.

> https:///auth/realms/master/protocol/openid-connect/login-status-iframe.html/init?client_id=security-admin-console&origin=https%3A%2F%2F

The only time I have seen urls like that being formed is when there is an invalid proxy header. You do not have it shown, but what is your proxy-headers setting, and can you provide what the proxy is setting?

**Comment by shawkins** [comment]
It would be clearer to provide the show-config output. I'm still uncertain if you are setting the proxy-headers option - if not, then the behavior you are seeing is not dependent on header values at all.

If you are setting the proxy-headers option, then providing the nginx config or an example of the header values it is setting would be good.

**Comment by Malikhaseebtahir** [comment]
Same is happening on my side.
Keycloak 26.0.5?

**Comment by jwklijnsma** [comment]
@Malikhaseebtahir what for env vars you have and do you have nginx-ingress?
